.github/workflows/quick-jobs.yml

```diff
@@ -62,6 +66,10 @@ jobs:
         with:
           path: ~/.cabal/store
           key: linux-store-doctest
+      - name: ghcup
+        run: |
+          ghcup install ghc recommended
+          ghcup set recommended
```

Comment:
you can do it in one step with `ghcup install ghc recommended --set`

Reply:
Noted. Let's keep it more verbose for now, while we start the overhaul.